Makefile

```diff
@@ -188,142 +188,26 @@ before-pr: vet test
-		--concurrent ${concurrent} \
-		--timeout ${timeout} \
-		--ci ${ci} \
-		--operator_image $(operator_image) \
-		--operator_tag $(operator_tag) \
-		--operator_yaml_uri ${operator_yaml_uri} \
-		--cli_path ${cli_path} \
-		--services_image_registry ${services_image_registry} \
-		--services_image_namespace ${services_image_namespace} \
-		--services_image_name_suffix ${services_image_name_suffix} \
-		--services_image_version ${services_image_version} \
-		--data_index_image_tag ${data_index_image_tag} \
-		--explainability_image_tag ${explainability_image_tag} \
-		--jobs_service_image_tag ${jobs_service_image_tag} \
-		--management_console_image_tag ${management_console_image_tag} \
-		--task_console_image_tag ${task_console_image_tag} \
-		--trusty_image_tag ${trusty_image_tag} \
-		--trusty_ui_image_tag ${trusty_ui_image_tag} \
-		--runtime_application_image_registry ${runtime_application_image_registry} \
-		--runtime_application_image_namespace ${runtime_application_image_namespace} \
-		--runtime_application_image_name_prefix ${runtime_application_image_name_prefix} \
-		--runtime_application_image_name_suffix ${runtime_application_image_name_suffix} \
-		--runtime_application_image_version ${runtime_application_image_version} \
-		--custom_maven_repo $(custom_maven_repo) \
-		--maven_mirror $(maven_mirror) \
-		--build_image_registry ${build_image_registry} \
-		--build_image_namespace ${build_image_namespace} \
-		--build_image_name_suffix ${build_image_name_suffix} \
-		--build_image_version ${build_image_version} \
-		--build_s2i_image_tag ${build_s2i_image_tag} \
-		--build_runtime_image_tag ${build_runtime_image_tag} \
-		--examples_uri ${examples_uri} \
-		--examples_ref ${examples_ref} \
-		--namespace_name ${namespace_name} \
-		--load_factor ${load_factor} \
-		--container_engine ${container_engine} \
-		--domain_suffix ${domain_suffix} \
-		--image_cache_mode ${image_cache_mode} \
-		--http_retry_nb ${http_retry_nb} \
-		--olm_namespace ${olm_namespace} \
-		--operator_installation_source ${operator_installation_source} \
-		--operator_catalog_image ${operator_catalog_image} \
-		$${opts_str}
+run-tests: download-kogito-operator-test-makefile
+	@(cd $(TEST_DIR) && $(MAKE) $@)
 
 .PHONY: run-smoke-tests
-run-smoke-tests:
-	make run-tests smoke=true
+run-smoke-tests: download-kogito-operator-test-makefile
+	@(cd $(TEST_DIR) && $(MAKE) $@)
 
 .PHONY: run-performance-tests
-run-performance-tests:
-	make run-tests performance=true
\ No newline at end of file
+run-performance-tests: download-kogito-operator-test-makefile
+	@(cd $(TEST_DIR) && $(MAKE) $@)
+
+.PHONY: build-examples-images
+build-examples-images: download-kogito-operator-test-makefile
+	@(cd $(TEST_DIR) && $(MAKE) $@)
+
+download-kogito-operator-test-makefile:
+	$(shell wget https://raw.githubusercontent.com/kiegroup/kogito-operator/$$(cat go.mod | grep 'github.com/kiegroup/kogito-operator' | awk -F'-' '{print $$4}')/test/Makefile -O test/Makefile)
```

Comment:
Waiting for https://github.com/kiegroup/kogito-operator/pull/821 to be merged to check the end to end on make command